src/test/scala/fi/oph/koski/migri/MigriSpec.scala

```diff
@@ -16,7 +16,7 @@ import org.scalatest.matchers.should.Matchers
 
 class MigriSpec extends AnyFreeSpec with KoskiHttpSpec with OpiskeluoikeusTestMethodsAmmatillinen with Matchers with DirtiesFixtures {
 
-  val user = MockUsers.luovutuspalveluKäyttäjä
```

Comment:
pitäisikö/voisikohan tämän MockUsers.luovutuspalveluKäyttäjä-käyttäjän (ja mahdollisesti jotkut sen käyttämistä rooleista) poistaa nyt kokonaan, vai vieläkö jää jokin viranomaistaho, joka sitä käyttää?

Eka kohta mihin törmäsin koodihaulla tämän osalta oli rikkinäinen testi, jossa käytetään testin kuvauksen kanssa ristiriitaista käyttäjätunnusta, koska valviralle on nykyään omansakin...

    "Kutsuminen vaatii VALVIRA-käyttöoikeuden" in {
      getHetu(KoskiSpecificMockOppijat.amis.hetu.get, user = MockUsers.luovutuspalveluKäyttäjä) {
        verifyResponseStatus(403, KoskiErrorCategory.forbidden())
      }
    }